assignment4.Rmd

```diff
@@ -105,14 +121,45 @@ sweep(loadings, 2, colSums(loadings), "/") #sweep() computes each row as a propo
 
 #Now examine your components and try to come up with substantive descriptions of what some might represent?
 
+# For PC1, the average number of hints and the average number of attempts are the most important elements. For PC2, the number of problems a student has answered and the percentage of problems a student has answered are the most important elements. For PC3, the average confidence each student has and the number of problems the student has attemped are the most important elements. For PC4, the number of problems a student has answered and the average confidence are the most important elements. For PC5, the number of problems the student has attempted and the average number of attempts are the most important elements. For PC6, the percentage of problems a student has answered correctly prior to this session and the average number of hints a student asked are the most important elements.
+
```

Comment:
Appreciate this detailed interpretation. Look at the combination of variables and pay attention to variables with the coefficient (in the eigenvector) larger than 0.2 (absolute value).